routing/router.go

```diff
@@ -1149,22 +1149,4 @@ func (r *ChannelRouter) SendPayment(payment *LightningPayment) ([32]byte, *Route
 	return preImage, route, nil
 }
 
-// TopologyClient...
-// TODO(roasbeef): put in discovery package?
-type TopologyClient struct {
-}
-
-// TopologyChange...
-type TopologyChange struct {
-	NewNodes    []*channeldb.LinkNode
-	NewChannels []*channeldb.ChannelEdge
-}
-
-// notifyTopologyChange...
-func (r *ChannelRouter) notifyTopologyChange() {
-}
-
-// SubscribeTopology....
-func (r *ChannelRouter) SubscribeTopology() (*TopologyClient, error) {
-	return nil, nil
-}
+// TODO(roasbeef): create TopologyClient
```

Comment:
As of the latest master, this TODO can be removed as the `TopologyClient` is implemented now.